src/Plugins/PluginInfo.php

```diff
@@ -94,10 +94,10 @@ protected function parsePluginInfo()
         $info = json_decode(file_get_contents($composer_json), true);
 
         if (!$info) {
-            throw new TerminusException('The file "{file}" is not a valid', ['file' => $composer_json]);
+            throw new TerminusException('The file "{file}" does not contain valid json', ['file' => $composer_json]);
         }
 
-        if (!$info['type'] || $info['type'] !== 'terminus-plugin') {
+        if (!isset($info['type']) || $info['type'] !== 'terminus-plugin') {
```

Comment:
A point of curiosity: When is this true?

`($info['type'] !== 'terminus-plugin') != ($info['type'] != 'terminus-plugin')`